tribute.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Document1</title>
```

Comment:
You are doing great so far. Please permit me to mention here that, your document is missing a proper document title. I believe it is always good practice to give your document a descriptive title  to tell your visitors what sort of information may be found on your website,